src/mame/yamaha/ymdx100.cpp

```diff
@@ -92,20 +306,264 @@ void yamaha_dx100_state::mem_map(address_map &map)
 }
 
 static INPUT_PORTS_START(dx100)
+	PORT_START("P2")
+	// TODO should 0x02, 0x04, 0x10, and 0x80 be listed here? they should be handled by the other interconnections in this file
+	// TODO if so, verify the active states of the MIDI ports
+	PORT_BIT( 0x01, IP_ACTIVE_LOW, IPT_OUTPUT )   PORT_NAME("LED") PORT_WRITE_LINE_MEMBER(yamaha_dx100_state, led_w)
+	PORT_BIT( 0x02, IP_ACTIVE_HIGH, IPT_UNUSED )  // tied to ground
+	PORT_BIT( 0x04, IP_ACTIVE_HIGH, IPT_CUSTOM )  // 500khz clock
+	PORT_BIT( 0x08, IP_ACTIVE_HIGH, IPT_CUSTOM )  PORT_CUSTOM_MEMBER(yamaha_dx100_state, midi_in_r)
+	PORT_BIT( 0x10, IP_ACTIVE_HIGH, IPT_OUTPUT )  // MIDI out
+	PORT_CONFNAME( 0x20, 0x00, "Foot Switch Connected?" )
```

Comment:
This can be simplified to just "Foot Switch"

Reply:
Done